### ISSUE TITLE
Create a jank REPL server

### ISSUE THREAD

**Issue by jeaye** [jank-lang/jank#23]
We need to figure out what goes into making editor-friendly REPL servers and cook one up. I think it makes the most sense to write it in jank itself, to dogfood sufficiently. I don't *think* existing Clojure REPL servers can work, but this should also be investigated.

As I intend for jank to be a monorepo of official tools, this project should be merged into main eventually.

This is blocked on jank being usable enough to write multi-file projects (needs require/class paths/etc) and work with native boxes (sockets, mainly).

**Comment by danielmartincraig** [comment]
It looks like this issue is no longer blocked by #31 , is that correct?  I'm interested in exploring this

**Comment by danielmartincraig** [comment]
🎉
